php/libraries/CouchDB.class.inc

```diff
@@ -118,6 +118,9 @@ class SocketWrapper
                     "Could not read from CouchDB server at $this->host:$this->port"
                 );
             }
+            while (!feof($this->socket)) {
+                $status .= fgets($this->socket);
```

Comment:
I think this should still check if the return value is === false and throw an exception (in case something exceptional happens between the first and subsequent reads). And at that point, you can just have the while loop without the initial read outside of the loop.